packages/esm-appointments-app/src/appointments/common-components/checkin-button.component.tsx

```diff
@@ -33,10 +31,7 @@ const CheckInButton: React.FC<CheckInButtonProps> = ({ appointment, patientUuid
                     to: checkInButton.customUrl,
                     templateParams: { patientUuid: appointment.patient.uuid, appointmentUuid: appointment.uuid },
                   })
-                : launchOverlay(
-                    t('patientSearch', 'Patient search'),
-                    <VisitForm patientUuid={patientUuid} appointment={appointment} />,
-                  )
+                : launchWorkspace('start-visit-workspace-form', { patientUuid: patientUuid })
```

Comment:
Instead of using the custom visit form defined in the appointments app, we use the new workspace functionality to launch the actual Visit Form defined in the patient chart.

Note that we will want to set the prop "showPatientHeader" to "true" once https://openmrs.atlassian.net/browse/O3-3157 is merged in.